QUDAConfig.cmake.in

```diff
@@ -25,12 +33,18 @@ if( QUDA_QIO )
 endif()
 
 if( QUDA_OPENMP )
-	find_dependency(OpenMP REQUIRED)
+  find_dependency(OpenMP REQUIRED)
 endif()
 
+if( QUDA_TARGET_CUDA )
+  include(${CMAKE_CURRENT_LIST_DIR}/find_cuda_deps.cmake)
```

Comment:
no strong feelings but wonder if that name is too generic if we put quda specific things in here. But I guess `find_quda_target_<target>_dependencies.cmake` is quite long.